src/components/ArtistCard.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
+import Footer from './components/Footer';
+
+{
+  currentUser && (
+    <Route
+      path="/ArtistCard"
+      element={
+        <ArtistProtectedIndex
+          currentUser={currentUser}
+          artists={artists}
+        />
+      }
+    />
+  )
+}
```

Comment:
It is convention to only define routes in one location in your app. I would consider moving this to App.js